packages/core/src/__tests__/state-manager.test.ts

```diff
@@ -315,7 +315,9 @@ describe('anchor', () => {
         )
 
         // We fast-forward streamState2, because the commit is legit
-        expect(streamState2.state).toEqual(stream1.state)
+        expect(StreamUtils.serializeState(streamState2.state)).toEqual(
```

Comment:
How did this change to how we use IPFS result in a change that broke a comparison of Ceramic state objects?

Reply:
Improper coercion from Buffer to Uint8Arrays in a 2nd level dependency. Should be fixed.